charts/ztka/values.yaml

```diff
@@ -391,6 +405,12 @@ kratos:
           configMapKeyRef:
             name: kratos-config
             key: selfservice_flows_error_ui_url
+      - name: SELFSERVICE_FLOWS_LOGIN_AFTER_PASSWORD_HOOKS_0_CONFIG_URL
```

Comment:
we should add it back when we upgrade kratos and use this approach

Reply:
Will remove for this release as discussed